scripts/normalize-by-median.py

```diff
@@ -270,6 +311,26 @@ def get_parser():
     return parser
 
 
+def CheckpointCountingTable(input_filenames, freq, ht, savename):
+    """
+    Generator/context manager to progressively save counting tables
+    """
+
+    for index, ifile in enumerate(input_filenames):
+        yield ifile
+        if (freq > 0 and index > 0 and index % freq == 0):
```

Comment:
can remove parantheses, and also just write:

     if freq and index and index % freq == 0: